script.js

```diff
@@ -1,4 +1,116 @@
+//-------------------------------------------Base Code---------------------------------------
+//array to store final number concatenated by the players
+var diceRolls = [];
+//random number rolled by dice 1
+var dice1 = 0;
+//random number rolled by dice 2
+var dice2 = 0;
+//number of turns completed. Each round there will be 2 turns since there are only 2 players
+var playerCount = 0;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  while (playerCount<2){
+    //generate dice roll numbers
+    if (input == ''){
+      dice1=Math.floor(Math.random()*6+1);
```

Comment:
For complex logic like getting a random num from 1-6, please put it in a helper function to avoid repeating this logic and inadvertently creating bugs.